src/ModalTitle.js

```diff
@@ -9,7 +9,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  componentClass: 'h4'
+  componentClass: 'h5'
```

Comment:
do the styles still require that this be a `h` tag? if not i think `div` is a better default, since it doesn't make an assumption about the document outline

Reply:
Agreed. Made `div` the default.